Formula/libiconv.rb

```diff
@@ -1,18 +1,10 @@
-class MacOSRequirement < Requirement
-  fatal true
-  satisfy(build_env: false) { OS.mac? }
-  def message
-    "is not needed on Linux, where iconv.h is provided by glibc"
-  end
-end
-
 class Libiconv < Formula
   desc "Conversion library"
   homepage "https://www.gnu.org/software/libiconv/"
   url "https://ftp.gnu.org/gnu/libiconv/libiconv-1.16.tar.gz"
   mirror "https://ftpmirror.gnu.org/libiconv/libiconv-1.16.tar.gz"
   sha256 "e6a1b1b589654277ee790cce3734f07876ac4ccfaecbee8afa0b649cf529cc04"
-  license "GPL-3.0"
+  license "GPL-3.0-or-later"
```

Comment:
While the `iconv` program is licensed under GPLv3+, the libraries/headers are licensed under older "GNU Library General Public License ... version 2 ..., or any later version."

I believe this is the previous name for LGPL, so probably need to use both since we install libraries, headers, and program.

```suggestion
  license all_of: ["GPL-3.0-or-later", "LGPL-2.0-or-later"]
```

https://www.gnu.org/software/libiconv/

> **Copyright**
> The `libiconv` and `libcharset` _libraries_ and their header files are under LGPL.
> The `iconv` _program_ is under GPL.

From `include/iconv.h`
```
   The GNU LIBICONV Library is free software; you can redistribute it
   and/or modify it under the terms of the GNU Library General Public
   License as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.
```

Reply:
The license is now fixed.  Thanks for catching that detail!